Resources/NetHookAnalyzer2/NetHookAnalyzer2/NetHookItemExtensions.cs

```diff
@@ -17,7 +18,13 @@ public static ListViewItem AsListViewItem(this NetHookItem item)
 				Text = item.Sequence.ToString(),
 			});
 
-			lvi.SubItems.Add(new ListViewItem.ListViewSubItem
+		    lvi.SubItems.Add(new ListViewItem.ListViewSubItem
+		    {
+		        Name = "Timestamp",
+		        Text = item.Timestamp.ToString(CultureInfo.InvariantCulture),
```

Comment:
Should this use `CurrentCulture` or `CurrentUICulture`, since it's being displayed to to the user?

Reply:
`CurrentUICulture` seems like a better option IMO.